pkg/processor/runtime/python/py/_nuclio_wrapper.py

```diff
@@ -169,23 +166,14 @@ async def serve_requests(self, num_requests=None):
                 self._unpacker = self._resolve_unpacker()
                 await self._on_serving_error(exc)
 
-            except asyncio.CancelledError:
-                self._logger.debug('Waiting for event message was interrupted by a signal')
-
             except Exception as exc:
                 await self._on_serving_error(exc)
 
             finally:
                 if self._is_drain_needed:
-                    result = self._call_drain_handler()
-                    if asyncio.iscoroutine(result):
-                        await result
-
+                    self._call_drain_handler()
                 if self._is_termination_needed:
-                    result = self._call_termination_handler()
-                    if asyncio.iscoroutine(result):
-                        await result
-                    break
```

Comment:
We can leave the `break` here, so the wrapper will exit after termination signal (and not accept any other events)